nekoyume/Assets/_Scripts/UI/Popup/BoosterPopup.cs

```diff
@@ -0,0 +1,135 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Nekoyume.BlockChain;
+using Nekoyume.Game;
+using Nekoyume.Game.Character;
+using Nekoyume.Model.Item;
+using Nekoyume.State;
+using TMPro;
+using UniRx;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace Nekoyume.UI
+{
+    public class BoosterPopup : PopupWidget
+    {
+        [SerializeField]
+        private Button cancelButton;
+
+        [SerializeField]
+        private Button submitButton;
+
+        [SerializeField]
+        private Slider apSlider;
+
+        [SerializeField]
+        private TMP_Text boostCountText;
+
+        [SerializeField]
+        private TMP_Text needAPText;
+
+        [SerializeField]
+        private TMP_Text ownAPText;
+
+        [SerializeField]
+        private Button boostPlusButton;
+
+        [SerializeField]
+        private Button boostMinusButton;
+
+        private Stage _stage;
+        private Player _player;
+        private List<Costume> _costumes;
+        private List<Equipment> _equipments;
+        private List<Consumable> _consumables;
+
+        protected override void Awake()
+        {
+            base.Awake();
+
+            cancelButton.OnClickAsObservable().Subscribe(_ => Close()).AddTo(gameObject);
+            submitButton.OnClickAsObservable().Subscribe(_ => BoostQuest()).AddTo(gameObject);
+            boostPlusButton.OnClickAsObservable().Subscribe(_ => apSlider.value++);
+            boostMinusButton.OnClickAsObservable().Subscribe(_ => apSlider.value--);
+        }
+
+        public void Show(Stage stage, List<Costume> costumes, List<Equipment> equipments, List<Consumable> consumables)
+        {
+            _stage = stage;
+            _costumes = costumes;
+            _equipments = equipments;
+            _consumables = consumables;
+            _player = _stage.GetPlayer(Vector3.zero);
+
+            ReactiveAvatarState.ActionPoint.Subscribe(value =>
+            {
+                apSlider.maxValue = value / 5 >= 12 ? 12 : value / 5;
```

Comment:
5 should refer to the action point on the stage sheet.
12 seems to be better to use by declaring it as const somewhere.